lib/chef/audit/runner.rb

```diff
@@ -96,6 +97,8 @@ def configure_rspec
         RSpec.configure do |c|
           c.color = Chef::Config[:color]
           c.expose_dsl_globally = false
+          separator = File::ALT_SEPARATOR ? File::ALT_SEPARATOR : File::SEPARATOR
+          c.backtrace_exclusion_patterns << Regexp.new(__FILE__.split(separator)[0..2].join(separator))
```

Comment:
I think we do want what you just described @lamont-granquist - on Windows we don't want to display any backtraces from `C:\opscode\chef\**\*`